_docs/_latest/develop/website.md

```diff
@@ -0,0 +1,176 @@
+4. [Install Jekyll](#install-jekyll)
+5. [Fetch, Build, and Run Forseti Web Code](#fetch-code)
+
+## Step by Step Instructions
+
+### Install Docker {#install-docker}
+
+```bash
+# Remove old docker-engine package (if installed)
+sudo apt-get remove docker docker-engine docker.io
+
+sudo apt-get update
+sudo apt-get install docker-ce
+```
+
+NOTE: If Docker is not available in the repository, follow
+the steps [here from the Docker
+website](https://docs.docker.com/install/linux/docker-ce/ubuntu/#set-up-the-repository)
+to add Docker to the package index.
+
+NOTE: If the previous command fails even after ensuring that Docker
+is in the package repository list, you may need to clear your
+docker lib (rm -rf /var/lib/docker) as well. Doing this will delete
+all images on disk. Ensure that they are backed up somewhere
+if you don't want this to happen.
+
+In order to avoid permission issues later on, add your user to the
+``docker`` group:
+
+```bash
+sudo adduser $USER docker
+```
+
+Before continuing, test that Docker is running correctly by
+doing the following:
+
+```bash
+docker run hello-world
+```
+
+### Install Ruby & RubyGems {#install-ruby}
+
+To install Ruby and RubyGems, execute the following:
+
+```bash
+sudo apt-get install ruby ruby-dev
+```
+
+### Install Bundler {#install-bundler}
+
+```bash
+gem install bundler
+```
+
+You may encounter the following error:
+
+```bash
+ERROR:  While executing gem ... (Gem::FilePermissionError)
+You don't have write permissions for the /var/lib/gems/2.3.0 directory.
+```
+
+If you do, the solution is [here](https://stackoverflow.com/a/47207118/1783829)
+and is listed below:
+
+```bash
+cd /var/lib
+sudo chmod -R a+w gems/
+```
+
+If you get the error again with ``You don't have write permissions for the
+/var/lib/bin`` the solution is listed below:
+
+```bash
+cd /usr/local/
+sudo chmod -R a+w gems/
+```
+
+### Install Jekyll {#install-jekyll}
+
+Install Jekyll by executing the following command:
+
+```bash
+gem install jekyll
+```
+
+If you run into the following error
+
+```bash
+ERROR: Error installing jekyll:
+ERROR: Failed to build gem native extension.
+...
+mkmf.rb can't find header files for ruby at /usr/lib/ruby/include/ruby.h
+```
+
+the root issue is that Jekyll has dependencies with Ruby on Linux that are
+not installed by default such as ``ruby-dev``. This should not happen
+if you installed [Ruby per above](#install-ruby).
+
+Additional context [here](https://github.com/jekyll/jekyll-help/issues/209).
+
+To install these additional dependencies, you need to execute the following:
+
+```bash
+sudo apt-get install ruby ruby-dev
+```
+
+With the dependencies now present, try installing Jekyll again:
+
+```bash
+gem install jekyll
+```
+
+### Fetch, Build, and Run Forseti Web Code {#fetch-code}
+
+#### Fetch Code
+Create a folder for the Forseti website and pull down the code.
+
+```bash
+mkdir forseti-web
+cd forseti-web/
+user@host:~/forseti-web$ git init
```

Comment:
Please remove `user@host:~/forseti-web$` to be consistent on how the shell command is shown.

Reply:
I added ``user@host:~/forseti-web$`` since it makes clear that the subsequent commands actually need to be executed within the ``forseti-web`` folder for the development server to work right. If we don't have ``user@host:~/forseti-web$``, the location to execute the commands is ambiguous.

Reply:
Okay, that makes sense now.